package.json

```diff
@@ -99,7 +99,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn tslint --fix && yarn format && yarn tslint",
+      "pre-commit": "yarn format && yarn tslint",
```

Comment:
Can we revert this change?

Reply:
fixed and removed duplicate entries in the file